### ISSUE TITLE
quotemark won't run if avoid-escape specified first

### ISSUE THREAD

**Issue by slaymaker1907** [palantir/tslint#3067]
### Bug Report

- __TSLint version__: 5.5.0
- __TypeScript version__: 2.4.2
- __Running TSLint via__: (pick one) CLI

#### TypeScript code being linted

```ts
import { Range } from './password';
```
with `tslint.json` configuration:

```json
{
    "rules": {
        "class-name": true,
        "comment-format": [
            true,
            "check-space"
        ],
        "indent": [
            true,
            "spaces"
        ],
        "no-eval": true,
        "no-internal-module": true,
        "no-trailing-whitespace": true,
        "no-unsafe-finally": true,
        "no-var-keyword": true,
        "one-line": [
            true,
            "check-open-brace",
            "check-whitespace"
        ],
        "quotemark": [
            true,
            "avoid-escape",
            "double",
            "jsx-double"
        ],
        "semicolon": [
            true,
            "always"
        ],
        "triple-equals": [
            true,
            "allow-null-check"
        ],
        "typedef-whitespace": [
            true,
            {
                "call-signature": "nospace",
                "index-signature": "nospace",
                "parameter": "nospace",
                "property-declaration": "nospace",
                "variable-declaration": "nospace"
            }
        ],
        "variable-name": [
            true,
            "ban-keywords"
        ],
        "whitespace": [
            true,
            "check-branch",
            "check-decl",
            "check-operator",
            "check-separator",
            "check-type"
        ]
    }
}
```

#### Actual behavior

Single quotes are not reported as an error.

#### Expected behavior

Single quotes should be reported as an error. It looks like "double" | "single" must be specified before "avoid-escape" otherwise it won't trigger.

**Comment by ajafff** [comment]
Closing this as it's a duplicate of #3063
